chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -18,20 +18,40 @@ import { defaultWebChatContainerStatefulProps } from "../../webchatcontainerstat
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const prepareEndChat = async (props: ILiveChatWidgetProps, chatSDK: any, state: ILiveChatWidgetContext, dispatch: Dispatch<ILiveChatWidgetAction>, setAdapter: any, setWebChatStyles: any, adapter: any, uwid: string) => {
     try {
+        // Use Case: If call is ongoing, end the call by simulating end call button click
+        const voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
```

Comment:
Is this adding a lot of time? We don't want to degrade performance on endChat, since this seems to be running on every endChat call

Reply:
why not to extract it and make it non blocking?, just call it and let it do its thing in the background

Reply:
Have not noticed a performance impact during testing.

Extracted call to another function.